client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -200,6 +202,18 @@ export default function CompositeCheckout( {
 		[ reduxDispatch ]
 	);
 
+	const checkoutFlow: string = useMemo( () => {
+		if ( isLoggedOutCart ) {
+			if ( isJetpackCheckout ) {
+				return isUserComingFromLoginForm
+					? 'jetpack_site_only_coming_from_login'
+					: 'Jetpack_site_only';
```

Comment:
Tiny consistency issue: the first letter is uppercase here.

Reply:
Oh yes, good catch! Thank you!